Typescript errors

### Current Behavior

I get the following typescript errors:

```
node_modules/pixi.js/pixi.js.d.ts:15554:37 - error TS1016: A required parameter cannot follow an optional parameter.

15554         constructor(width?: number, points: PIXI.Point[]);
                                          ~~~~~~

node_modules/pixi.js/pixi.js.d.ts:21875:35 - error TS2314: Generic type 'Array<T>' requires 1 type argument(s).

21875         function removeItems(arr: Array, startIdx: number, removeCount: number): void;
                                        ~~~~~


Found 2 errors.
```

They are new in 5.0.0-rc.3

### Environment

- **`pixi.js` version**: _5.0.0-rc.3_

--- a/packages/mesh-extras/src/geometry/RopeGeometry.js
+++ b/packages/mesh-extras/src/geometry/RopeGeometry.js
@@ -18,7 +18,7 @@ export default class RopeGeometry extends MeshGeometry
 {
     /**
      * @param {number} [width=200] - The width (i.e., thickness) of the rope.
-     * @param {PIXI.Point[]} points - An array of {@link PIXI.Point} objects to construct this rope.
+     * @param {PIXI.Point[]} [points] - An array of {@link PIXI.Point} objects to construct this rope.
      */
     constructor(width = 200, points)
     {

--- a/packages/utils/src/data/removeItems.js
+++ b/packages/utils/src/data/removeItems.js
@@ -3,7 +3,7 @@
  *
  * @function removeItems
  * @memberof PIXI.utils
- * @param {Array} arr Array to remove elements from
+ * @param {Array<any>} arr Array to remove elements from
  * @param {number} startIdx starting index
  * @param {number} removeCount how many to remove
  */

